reference/constraints/Image.rst

```diff
@@ -392,4 +406,16 @@ Portrait oriented images are not allowed``
 
 The error message if the image is portrait oriented and you set `allowPortrait`_ to ``false``.
 
+corruptedMessage
+~~~~~~~~~~~~~~~~
+
+.. versionadded:: 3.1
+     The ``corruptedMessage`` option was introduced in Symfony 3.1.
+
+**type**: ``string`` **default**: ``The image file is corrupted.``
+
+The error message when the `detectCorrupted`_ option is enabled and the image
```

Comment:
I was to unsure to commit this, but shouldn't we use `if` here instead of `when`?